src/Nethermind/Nethermind.Core/Specs/IEip1559Spec.cs

```diff
@@ -22,4 +22,26 @@ public interface IEip1559Spec
         public UInt256 BaseFeeMaxChangeDenominator { get; }
         public long ElasticityMultiplier { get; }
     }
+
+    public sealed class Eip1559Spec : IEip1559Spec
```

Comment:
This class is used to wrap an existing `IEip1559Spec` and override certain parameters.

A more elegant solution might be possible and I'm open to suggestions.